string command  # "open" or "close"
---
bool success
string message
